assignments/lambda-classes.js

```diff
@@ -4,8 +4,23 @@ class Person{
         this.name= baseClass.name;
```

Comment:
Your commits are so meticulous and well-documented that your bosses are going to love you. Keep it up!